auxx/src/Command/Proc.hs

```diff
@@ -51,68 +52,80 @@ import           Lang.Name (Name)
 import           Lang.Value (AddKeyParams (..), AddrDistrPart (..), GenBlocksParams (..),
                              ProposeUpdateParams (..), ProposeUpdateSystem (..),
                              RollbackParams (..), SendMode (..), Value (..))
-import           Mode (AuxxMode, CmdCtx (..), deriveHDAddressAuxx, getCmdCtx, makePubKeyAddressAuxx)
+import           Mode (MonadAuxxMode, CmdCtx (..), deriveHDAddressAuxx, getCmdCtx, makePubKeyAddressAuxx)
 import           Repl (PrintAction)
 
 createCommandProcs ::
-       (HasConfigurations, HasCompileInfo)
-    => PrintAction AuxxMode
-    -> SendActions AuxxMode
-    -> [CommandProc AuxxMode]
-createCommandProcs printAction sendActions = fix $ \commands -> [
-
-    CommandProc
+       forall m. (HasCompileInfo, MonadIO m, CanLog m, HasLoggerName m)
+    => Maybe (Dict HasConfigurations)
```

Comment:
I don't see a single command that uses `needsConfigurations` but doesn't need `MonadAuxxMode`. Since `MonadAuxxMode` already includes `HasConfigurations`, a separate `Dict` for `HasConfigurations` seems redundant to me.